cstore_reader.c

```diff
@@ -684,6 +699,13 @@ LoadStripeSkipList(FILE *tableFile, StripeMetadata *stripeMetadata,
 		ColumnBlockSkipNode *columnSkipList = NULL;
 		uint32 blockIndex = 0;
 
+		/* no need to create ColumnBlockSkipList if the column is not selected */
+		if (!projectedColumnMask[columnIndex] && columnIndex > 0)
```

Comment:
Instead of doing `columnIndex > 0` here and later, can we just add `rowCount` to `struct StripeSkipList` and populate it at `LoadStripeSkipList()` and remove `StripSkipListRowCount()`? This way reader of code won't wonder why `columnIndex > 0` is needed.